tests/e2e/UpdateProfile.js

```diff
@@ -0,0 +1,85 @@
+import 'testcafe';
+import LayoutPage from './page-objects/Layout';
+import LogIn from './page-objects/Login';
+import Notifications from './page-objects/Notifications';
+import UpdateProfile from './page-objects/UpdateProfile';
+import HomePage from './page-objects/Homepage';
+
+const logIn = new LogIn();
+const layoutPage = new LayoutPage();
+const notifications = new Notifications();
+const updateProfile = new UpdateProfile();
+const homePage = new HomePage();
+const uploadedPreviewImages = layoutPage.Content.find(
```

Comment:
probably candidate to move to UpdateProfile page object

Reply:
you're right, fixed